README.md

```diff
@@ -70,19 +67,16 @@ After the migration has been published you can create the role- and permission-t
 running the migrations:
 
 ```bash
-php artisan migrate
+$ php artisan migrate
 ```
 
 You can publish the config-file with:
 ```bash
-php artisan vendor:publish --provider="Spatie\Permission\PermissionServiceProvider" --tag="config"
+$ php artisan vendor:publish --provider="Spatie\Permission\PermissionServiceProvider" --tag="config"
```

Comment:
Please don't start this with `$`, because it means triple-click to copy-and-paste the whole line will include the unnecessary `$`.

Same 5 lines up too.